apps/src/templates/instructions/RubricField.jsx

```diff
@@ -0,0 +1,92 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import color from '@cdo/apps/util/color';
+import Radium from 'radium';
+
+const styles = {
+  detailsArea: {
+    width: '100%'
+  },
+  rubricHeader: {
+    fontSize: 13,
+    marginLeft: 10,
+    color: color.black,
+    fontFamily: '"Gotham 5r", sans-serif'
+  },
+  performanceLevelHeader: {
+    display: 'flex',
+    justifyContent: 'flex-start',
+    flexDirection: 'row',
+    padding: '4px 10px',
+    ':hover': {
+      border: 'solid 1px' + color.light_cyan,
+      borderRadius: 10
+    }
+  },
+  performanceLevelHeaderSelected: {
+    display: 'flex',
+    justifyContent: 'flex-start',
+    flexDirection: 'row',
+    backgroundColor: color.lightest_cyan,
+    borderRadius: 10,
+    padding: '4px 10px',
+    ':hover': {
+      border: 'solid 1px' + color.light_cyan,
+      borderRadius: 10
+    }
+  }
+};
+
+const rubricLevelHeaders = {
+  exceeds: 'Exceeds',
+  meets: 'Meets',
+  approaches: 'Approaches',
+  noEvidence: 'No Evidence'
+};
+
+class RubricField extends Component {
+  static propTypes = {
+    showFeedbackInputAreas: PropTypes.bool,
+    rubricLevel: PropTypes.string,
+    rubricValue: PropTypes.string,
+    disabledMode: PropTypes.bool,
+    onChange: PropTypes.func,
+    currentlyChecked: PropTypes.bool
+  };
+
+  handleRubricChange = event => {
+    this.props.onChange(event.target.value);
+  };
+
+  render() {
+    return (
+      <div
+        style={
+          this.props.currentlyChecked
+            ? styles.performanceLevelHeaderSelected
+            : styles.performanceLevelHeader
+        }
+      >
+        {this.props.showFeedbackInputAreas && (
+          <input
+            type={'checkbox'}
+            id={this.props.rubricLevel + '-input'}
```

Comment:
because element ids are global to the document, it's important for them to be specific. I would recommend calling these:
```js
            id={`rubric-input-${this.props.rubricLevel}`}
```
this will also make it a bit easier for others to find where these ids are coming from when they search the codebase.

one alternative is to use css classnames rather than ids for this purpose, but this is just an option and I think ids (once made more specific) are fine here.